### ISSUE TITLE
Is there a way to turn off synchronization without losing clipboard contents?

### ISSUE THREAD

**Issue by richlux** [hluk/CopyQ#1974]
I have a number of CopyQ tabs and I am currently synchronizing them to a shared folder.  I would like to temporarily stop synchronizing, but when I uncheck the Synchronize option in the Items preferences menu, I lose all of my clipboard items.  Is there a way to turn of synchronization without losing the clipboard items?

![image](https://user-images.githubusercontent.com/25157703/165792336-4349ca61-0be0-4b16-8ff9-a59aa01cf39c.png)

Thanks,
Rich

**Comment by robertstrom** [comment]
Back up CopyQ first and then turn off sync and then restore. That is what I would do anyway. I also have backup set to run every time that CopyQ is closed.

**Comment by richlux** [comment]
Thanka, That worked, but when I did the import after I turned off synchronization, it imported the data into new tabs so I had to delete the old tabs and rename the new tabs. I believe that is how import is supposed to work, but it would be nice to give an option to overwrite existing tabs. 

Rich

**Comment by robertstrom** [comment]
>  it imported the data into new tabs so I had to delete the old tabs and rename the new tabs. I believe that is how import is supposed to work, but it would be nice to give an option to overwrite existing tabs.

I totally agree! It would be a nice feature to have the option of deciding how to handle the import.

I guess it has become second nature to me to just start from scratch when doing an import from a backup or, what I do more often, is just backup the directory structure that I mentioned above (depends on what OS you are running), the **copyq** directory in your **~/.config/** or your **C:\Users<username>\AppData\Roaming\**. If you shutdown CopyQ, copy this entire copyq directory structure, and then just copy it to whatever CopyQ instance and restart CopyQ you will be good to go. This directory has all of the settings and data.

I use a ton of virtual machines and I am installing CopyQ and populating it with data on new virtual machines on a fairly regular basis so I am quire familiar with the process ;-)

IMHO there needs to a merge capability so we can combine the contents of multiple CopyQ data sets into one. Setting up sync for each and every CopyQ tab is cumbersome to say the least. I have over 100 tabs and I have CopyQ on 2 primary Macs, 1 physical Windows machine and multiple Windows virtual machines, and 2 physical Linux machines and multiple Linux virtual machines. I can be using any one of these systems at any time and copy an entry that I want to save into some tab (it could even be a new tab which does not exist on another computer).

Ultimately I would love to be able to have one CopyQ directory structure that is considered the primary data source and have a programmatic function that I can tell to look at another directory structure and to merge the differences from the secondary directory structure to the primary directory structure.

I may have to do this a few times periodically but I would ultimately end up with one directory structure that has one complete data set.

Either what I have described above or a new, easier to manage, sync capability.

Glad that I was able to help!

Robert
